faithcomesbyhearing/dbp
# FCBHDBP-456 Migrate to swagger-php version 4

# Description
Updated swagger version to 4. It has moved some openApi annotations to support the latest version of the swagger library. it has moved some schemas definitions created inside methods to the class level.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-456

## How Do I QA This
1. You can import the new openApi.json file into postman to check that it is valid.
2. You should run the following command to generate the openApi.json file and it should work:
```shell
php artisan openApi:generate
```
